test/Serilog.Enrichers.ClientInfo.Tests/TestSetup/WebApi.cs

```diff
@@ -0,0 +1,52 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.HttpOverrides;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Serilog;
+using Serilog.Enrichers.ClientInfo.Tests;
+using Serilog.Events;
+
+Log.Logger = new LoggerConfiguration()
+    .Enrich.WithClientIp()
+    .Enrich.WithRequestHeader("X-Forwarded-For")
+    .WriteTo.Sink(new DelegatingSink(e => LogEvent = e))
+    .CreateLogger();
+
+var builder = WebApplication.CreateBuilder(args);
+builder.Host.UseSerilog();
+builder.Services
+    .AddHttpContextAccessor()
+    .Configure<ForwardedHeadersOptions>(
+        options =>
+        {
+            options.ForwardedHeaders = ForwardedHeaders.XForwardedFor;
+            options.KnownNetworks.Clear();
+            options.KnownProxies.Clear();
+        });
+
+var app = builder.Build();
+
+app
+    .UseForwardedHeaders()
+    .Use(
+        async (context, next) =>
+        {
+            app.Logger.LogInformation(
+                "HttpContext.Connection.RemoteIpAddress: {RemoteIpAddress}, HttpContext.Items[\"Serilog_ClientIp\"]: {Serilog_ClientIp}",
```

Comment:
Since there is no `Serilog_ClientIp` in `ClientIpEnricher` code, I believe we can safely remove it from here as well.